src/verilogAST.cpp

```diff
@@ -200,8 +200,7 @@ std::string Module::toString() {
   // emit body
   for (auto statement : body) {
     module_str +=
-        variant_to_string<Always *, StructuralStatement *, Declaration *>(
-            statement) +
+        variant_to_string<StructuralStatement *, Declaration *>(statement) +
         ";\n";
```

Comment:
When generating the string for a module, the `;` are inserted after every statement, so Declaration/Assign shouldn't generate the semicolon to be consistent, or this semicolon should be removed